src/routes/formsgSiteLaunch.ts

```diff
@@ -34,6 +38,7 @@ interface FormResponsesProps {
   primaryDomain?: string
   redirectionDomain?: string
   agencyEmail?: string
+  siteLaunchDetails?: string[] | string[][]
```

Comment:
could i ask why it is a nested array?

Reply:
if this is cos of the form responses then... :(

Reply:
Yes hahah let me put in a comment to address this query d5e31ec

Reply:
tbh we might wanna convert both `string[] | string[][]` into a representation that's easier to work with like an object with `source: string, target: string, type: A | CNAME` etc but it's not immediately required and seems to be more of an extension